dts/nxp/nxp_imx/rt/MIMXRT735SGAWAR-pinctrl.h

```diff
@@ -14,8 +14,8 @@
 	(((offset) & 0xFFF) << 20) |		\
 	(((mux) & 0xF) << 0))
 
-#define PMIC_I2C_SCL IOPCTL_MUX(1, 150, 0) /* PIO0_0 */
-#define PMIC_I2C_SDA IOPCTL_MUX(1, 151, 0) /* PIO0_0 */
```

Comment:
Did you rerun the script or was this manually fixed?

Reply:
I manually fixed the header files we already had upstream.